js/src/wp-seo-network-admin.js

```diff
@@ -0,0 +1,96 @@
+/* global wpseoNetworkAdminGlobalL10n, ajaxurl */
+
+import a11ySpeak from "a11y-speak";
+
+( function( $ ) {
+	"use strict";
+
+	/**
+	 * Displays given settings errors.
+	 *
+	 * @param {Object} settingsErrors The list of settings error objects.
+	 *
+	 * @returns {void}
+	 */
+	function displaySettingsErrors( settingsErrors ) {
+		var $heading = $( ".wrap > h1" );
```

Comment:
minor: this is used for both success and error messages, I'd consider to rename it